packages/insomnia-app/app/network/o-auth-2/grant-authorization-code.ts

```diff
@@ -37,8 +38,15 @@ export default async function(
   if (usePkce) {
     // @ts-expect-error -- TSCONVERSION
     codeVerifier = _base64UrlEncode(crypto.randomBytes(32));
-    // @ts-expect-error -- TSCONVERSION
-    codeChallenge = _base64UrlEncode(crypto.createHash('sha256').update(codeVerifier).digest());
+
+    if (pkceMethod == c.PKCE_CHALLENGE_S256){
+      // @ts-expect-error -- TSCONVERSION
+      codeChallenge = _base64UrlEncode(crypto.createHash('sha256').update(codeVerifier).digest());
+    }
+    else{
```

Comment:
```suggestion
    } else {
```

just fixing this via the GitHub UI in anticipation of the CI blocking on https://github.com/Kong/insomnia/pull/3859 which looks like it'll be merged right before this.